trillian.proto

```diff
@@ -217,8 +217,13 @@ message SignedLogRoot {
   int64 tree_size = 3;
   sigpb.DigitallySigned signature = 4;
 
-  int64 log_id = 5;
+  reserved 5; // log_id is associated with the public key that validates signature.
   int64 tree_revision = 6;
+  // key_hint is a hint to identify the public key for signature verification.
+  // key_hint is not authenticated and may be incorrect or missing, in which
+  // case all known public keys may be used to verify the signature.
+  // key_hint MAY contain the LogID encoded as a big endian 64 bit integer.
```

Comment:
Maybe add something about context: "When directly communicating with a Trillian gRPC server, the key_hint will typically contain the LogID encoded as a big-endian 64-bit integer; however, in other contexts the key_hint is likely to have different contents (e.g. it could be Trillian URL + tree ID, or it could be data derived from the public key itself)."

Reply:
Done